github-pr.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+
+/**
+ * Content script that runs in the context of github pull request web pages and
+ * adds "attach to bug" link.
+ */
+
+// Regexp to match against PR title
+var bugRe = /\b(ticket|bug|tracker item|issue)s?:? *([\d ,\+&#and]+)\b/i;
```

Comment:
An easy ES6-ification that you could do is stop using `var` and start using `let` and `const`. They provide a couple benefits, namely that scopes are saner (block level instead of function level), and that `const` lets you say that something won't change, which is nice for readability.